p2p/libp2p/networksharding/sortingList.go

```diff
@@ -33,3 +34,24 @@ func (sl *sortingList) Less(i int, j int) bool {
 func (sl *sortingList) Swap(i int, j int) {
 	sl.peers[i], sl.peers[j] = sl.peers[j], sl.peers[i]
 }
+
+// SortedPeers get the orted list of peers
+func (sl *sortingList) SortedPeers() []peer.ID {
+	sort.Sort(sl)
+	ret := make([]peer.ID, len(sl.peers))
+
+	for i, id := range sl.peers {
+		ret[i] = id.id
+	}
+	return ret
+}
+
+func (sl *sortingList) InShardCount() int {
```

Comment:
This sorting list knows how to count in shard peers. I think this should be move inside kadSharder and be unexported because it is not the sortingList's job.

Reply:
Done